service/handlers.go

```diff
@@ -150,7 +150,11 @@ func (h handler) createWorkflowFromGit(w http.ResponseWriter, r *http.Request) {
 
 	level.Debug(l).Log("message", "validating authorization header for create workflow from git")
 	ah := r.Header.Get("Authorization")
-	a := credentials.NewAuthorization(ah)
+	a, err := credentials.NewAuthorization(ah)
```

Comment:
is it possible that the NPE is happening here? I just checked this code is calling strings.SplitN on the header value, and I'm not sure if that handles when the string is 'nil' (when no Authorization header is present on the request)?

Reply:
Talked through this, should be handled